cypress/integration/dropdown.js

```diff
@@ -37,25 +37,14 @@ describe('Dropdown test', function() {
         });
 
         it('Should show all accounts as linked (except GitLab and Bitbucket)', function() {
-            cy
-                .get('h4').contains('GitHub').contains('Unlink Account')
-            cy
-                .get('h4').contains('Quay.io').contains('Unlink Account')
-            cy
-                .get('h4').contains('Bitbucket').contains('Link Account')
-            cy
-                .get('h4').contains('GitLab').contains('Link Account')
+            everythingOk();
         });
     });
     var everythingOk = function() {
-        cy
-        .get('h4').contains('GitHub').contains('Unlink Account')
-        cy
-        .get('h4').contains('Quay.io').contains('Unlink Account')
-        cy
-        .get('h4').contains('Bitbucket').contains('Link Account')
-        cy
-        .get('h4').contains('GitLab').contains('Link Account')
+            cy.get('#unlink-GitHub').should('be.visible')
```

Comment:
Maybe just reference the values from accounts.component.ts from here rather than relying on the bare strings matching?

Reply:
The IDs are dynamically generated.  I currently can't import ts files into Cypress.io .js files

Reply:
If you can't reference ts from js, maybe draw both from a properties file or similar?

Reply:
Had a very simple `export enum AccountInfo` file, didn't work.